vita3k/gui/src/about_dialog.cpp

```diff
@@ -48,7 +48,7 @@ static std::vector<const char *> contributors_list = {
     "yousifd", "Yunotchi"
 };
 
-static std::vector<const char *> supporters_list = {
+static constexpr std::array<const char *, 4> supporters_list = {
```

Comment:
```suggestion
static constexpr std::array supporters_list = {
```
I'm not sure if full type description is needed. I think shorter code is better and also there will be no need to count items in array manually

Reply:
Either is fine, but since it is written in most codes i searched, I'll leave it.

Reply:
I will follow the majority.
In addition, the list of contributors has been updated.